Clojure-Functions.md

```diff
@@ -0,0 +1,82 @@
+# Clojure - More on Functions
+Functions! They're pretty important. It's pretty difficult to do anything without
+a function. They're integral to any language, but especially Clojure, since it's
+a functional programming language that rejects object-oriented design. Let's
+learn some more about them!
+
+## Arity
+'Arity' refers to the number of arguments that your function expects.
```

Comment:
Nah. I tend to `code quote` things like variable names (`foo`, `bar`) or function/macro names (`inc`, `defn`). I just 'quoted' arity because it's a programming term, but not a term in the language, if that makes sense. There's no `arity` variable/function/macro.